cloud/datasync/example/index.ts

```diff
@@ -35,8 +35,8 @@ const router = middleware.createSyncExpressRouter();
 app.use('/', router);
 
 app.listen(3000, function() {
-  // tslint:disable-next-line:no-console
-  console.log('Example app listening on port 3000!');
+  logger.info('Example app listening on port 3000!',
+    {level: 'INFO', tag: 'cloud:datasync:example', src: 'index.ts'});
```

Comment:
No problem, this is mostly an issue with documentation, our users would have no information about what kind of data we're passing from our own modules into the logger, so their implementations can only go for that too. The `{level, tag, src}` tuple is just a convention in this very PR, that we'll need to watch out so we adhere to it ourselves.

Just want to point out that ts definitions for existing libs have a different goal than the design of the interfaces themselves.